ECMAScript/core/src/types.ts

```diff
@@ -8,3 +8,8 @@ export type Nullable<T> = { [K in keyof T]: T[K] | null }
 export type Mutable<Type> = { -readonly [Key in keyof Type]: Type[Key] }
 export type DeepPartial<T> = T extends object ? { [P in keyof T]?: DeepPartial<T[P]> } : T
 export type DeepReadonly<T> = T extends object ? { [P in keyof T]: DeepReadonly<T[P]> } : T
+declare global {
+  interface Window {
+    expo: any
```

Comment:
is there a more specific type for this?

Reply:
I don't think we really care until we use anything relate to `window.expo`, plus we might not even target expo app at all because we want the package to work with both expo app and pure react native app.